tests/e2e/standalone/run_template_test.go

```diff
@@ -0,0 +1,365 @@
+//go:build e2e || template
+// +build e2e template
```

Comment:
This test will run if either e2e or template tag is passed. Locally using one of the make commands only template need to be passed. In CI, e2e is the tag that is passed